### ISSUE TITLE
Concurrent Update Exceptions in MVRB Tree/Indices [moved]

### ISSUE THREAD

**Issue by lvca** [orientechnologies/orientdb#691]
This is [Issue 691](http://code.google.com/p/orient/issues/detail?id=691) moved from a Google Code project.
Added by 2012-01-25T16:28:37.000Z by [arichard...@gmail.com](http://code.google.com/u/104192970901926255849/).
Please review that bug for more context and additional comments, but update this bug.
 Closed (Fixed).

Original labels: Type-Defect, Priority-Medium, v1.0rc8
### Original description

```
OrientDB release? 
Svn 4654


<b>What steps will reproduce the problem?</b>

See this thread for test code (Luca reported that he had reproduced the problem):

http://groups.google.com/group/orient-database/browse_thread/thread/b931d65ae573e4de

In summary:

When running a test that involves repetitive insert/delete through a 
transaction where an index is present, an invalid 
ConcurrentModificationException error is thrown.  It seems to be 
related to something in the MVRBTree.
```
